opentelemetry-etw-logs/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "opentelemetry-etw-logs"
+description = "OpenTelemetry-Rust exporter to ETW"
+version = "0.1.0"
+edition = "2021"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-logs"
+repository = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-logs"
+readme = "README.md"
+rust-version = "1.65.0"
+keywords = ["opentelemetry", "log", "trace", "etw"]
+license = "Apache-2.0"
+
+[dependencies]
+tracelogging = "1.2.1"
+tracelogging_dynamic = "1.2.1"
+opentelemetry = { workspace = true, features = ["logs"] }
+opentelemetry_sdk = { workspace = true, features = ["logs"] }
+async-std = { version = "1.6" }
+async-trait = { version = "0.1" }
+chrono = { version = "0.4", default-features = false, features = ["std"] }
+serde_json = "1.0.113"
+
+[dev-dependencies]
+opentelemetry-appender-tracing = { workspace = true }
+tracing = { version = "0.1", default-features = false, features = ["std"] }
+tracing-core = "0.1.31"
+tracing-subscriber = { version = "0.3.0", default-features = false, features = [
+    "registry",
+    "std",
+] }
+microbench = "0.5"
```

Comment:
nit - Is there a particular reason we're leaning towards `microbench` rather than `criterion`, as `otel-rust` mainly sticks with `criterion`? Just curious, as it doesn’t necessarily mean we can’t explore using it here. :)